apps/site/src/views/Schedule/sections/ClipboardSchedule/ClipboardSchedule.tsx

```diff
@@ -27,21 +27,26 @@ const dateTimeFormat = new Intl.DateTimeFormat("en", {
 	minute: "numeric",
 });
 
+const weekdayFormat = new Intl.DateTimeFormat("en", {
+	weekday: "long",
+});
+
 function ClipboardSchedule({ schedule }: ClipboardScheduleProps) {
 	return (
 		<Container
 			as="section"
-			className={styles.clipboard + " px-0 position-relative"}
+			className={styles.clipboard + " px-0 pt-5 position-relative"}
 		>
 			<div className={styles.clip}>
 				<Image src={clip} alt="Clipboard clip" className={styles.clip} />
 			</div>
 			<Accordion defaultActiveKey="0" className={styles.accordion}>
 				{schedule.map((day, i) => (
-					<div key={i}>
-						<h2 className={styles.date}>
+					<div className="mt-5" key={i}>
+						<h2>{weekdayFormat.format(day[0].startTime)}</h2>
+						<h3 className="text-center mb-5">
```

Comment:
Chore: Using a stacked nested heading here doesn't quite make sense, especially when we're using `h3` for the accordion items. Bootstrap provides utility classes if you want just the heading style (e.g. `.h3`) or sizing (e.g. `.fs-4`)